app/domain/authentication/authn_jwt/jwt_fetch_signing_key_factory.rb

```diff
@@ -0,0 +1,65 @@
+module Authentication
+  module AuthnJwt
+    # Factory that returns the interface implementation of FetchSigningKey
+    class JwtFetchSigningKeyFactory
+      FETCH_SIGNING_KEY_METHODS = {
+        "provider-uri" => JwtFetchProviderUriSigningKey,
+        "jwks-uri" => JwtFetchJwksUriSigningKey
+      }
+
+      def signin_key_implementation(authenticator_input)
```

Comment:
is this the method that creates the `FetchSigningKey` object? can it be called `create` instead?